ch_backup/clickhouse/client.py

```diff
@@ -37,7 +37,7 @@ def settings(self):
         """
         return self._session.params
 
-    @retry(requests.exceptions.ConnectionError)
+    @retry((requests.exceptions.ConnectionError, ClickhouseError))
```

Comment:
Retry of a failed non-SELECT queries will cause unnecessary logs and longer fail time, in that case a check if query is a SELECT query can be added with retries on `ClickhouseSelectError` or something similar